pkg/domain/ecosystem/componentInstallation.go

```diff
@@ -0,0 +1,34 @@
+package ecosystem
+
+import (
+	"github.com/cloudogu/cesapp-lib/core"
+)
+
+// ComponentInstallation represents an installed or to be installed component in the ecosystem.
+type ComponentInstallation struct {
+	// Name is the name of the component, e.g. 'k8s-dogu-operator'.
+	// The name is also the ID of the component in the ecosystem as only one component with this name can be installed.
+	Name string
+	// DistributionNamespace is part of the address under which the component will be obtained. This namespace does NOT
+	// to be confused with the K8s cluster namespace.
+	DistributionNamespace string
```

Comment:
Here as well. I thought the `DistributionNamespace` would continue to be part of the `Name`.